### DISCUSSION TITLE
z/OS Open Source Guild Meeting #4 - November 30, 2022 at 11am EST

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#78] [Guild]
### Agenda

* Highlights in z/OS Open Source (10 mins)
  * Progress report: (https://github.com/ZOSOpenTools/meta/blob/main/docs/Progress.md)
  * Introducing git, bash, cmake, wget, ncdu and more
  * Notable zopen framework updates
* Introducing zopen-setup (Mike Fulton)
* An example of porting to z/OS (Haritha D)
* Our experience with upstreaming (Jeff Shimoda)
* Badge program discussion (5 mins)
* Feedback (5 mins)
* Polls and Open Discussion (10 mins)

### How to Join?
Join link: https://ibm.webex.com/ibm/j.php?MTID=m72857a0933fbcd1c58c116a13c9f1e7b 
Webinar number: 145 105 4264
Webinar password: QamxS8QX2m7 (72697879 from phones)

### Date and Time
* Nov. 30, 11am-12pm EST (8am PST)

### Recording/Slide/Poll/Chat Log
* [Powerpoint Slides](https://github.com/ZOSOpenTools/meta/files/10125732/Open.Source.Guild.November.2022_final.pptx)
* [Chat log](https://github.com/ZOSOpenTools/meta/files/10125770/chat-en_us.1.csv)
* [Poll](https://github.com/ZOSOpenTools/meta/files/10125780/poll-results.1.pdf)
* [Recording](https://ibm.box.com/s/xau73pn3emlf5nlyzn7aq198s11jj0w9)

**Comment by ijmitch** [comment]
@IgorTodorovskiIBM ... the [Recording](https://ibm.box.com/s/n11pidx5oi1feorxc4nif04kj4jwqvg4) link didn't work for me.

**Reply by EnsonoStevenPerva** [reply]
I am having the same issue with the recording. @IgorTodorovskiIBM

**Reply by IgorTodorovskiIBM** [reply]
Interesting, I have reshared it here: https://ibm.box.com/s/xau73pn3emlf5nlyzn7aq198s11jj0w9 .

**Reply by ijmitch** [reply]
what one worked for me - thanks @IgorTodorovskiIBM
